src/Parallel/Main.hpp

```diff
@@ -259,23 +268,18 @@ Main<Metavariables>::Main(CkArgMsg* msg) noexcept
       const_global_cache_proxy_.ckGetGroupID());
 
   tmpl::for_each<group_component_list>([
-    this, &the_parallel_components, &const_global_cache_dependency
+    this, &the_parallel_components, &items_from_options, &
+    const_global_cache_dependency
```

Comment:
This `&` formatting doesn't look right

Reply:
I agree, but it is what clang-format does 🤷‍♂